docs/Validation-and-Serialization.md

```diff
@@ -658,7 +658,92 @@ fastify.setErrorHandler(function (error, request, reply) {
 })
 ```
 
-If you want custom error response in schema without headaches and quickly, you can take a look at [here](https://github.com/epoberezkin/ajv-errors)
+If you want custom error response in schema without headaches and quickly, you can take a look at [`ajv-errors`](https://github.com/epoberezkin/ajv-errors).
+
+Below is an example showing how to add custom error messages for each property of a schema by supplying a custom AJV instance. Inline comments describe how to configure the schema to show a different error message for each case:
```

Comment:
I would add this example [link](https://github.com/fastify/example/tree/master/validation-messages#example-of-custom-validation-errors-using-ajv-and-fastify-v2) as "quick play".

Reply:
You mean remove the example snippet from here and only have the link to example?

Reply:
No, it is ok the example here 👍

I would like to give to that example more visibility